model_compiler/tests/model_compiler/compilers/test_onnx_model_to_tensorrt_model.py

```diff
@@ -60,7 +72,32 @@ def _make_onnx_model(func, batch_size_1, batch_size_2):
                                session=session)
     )
 
-    return onnx_compiler.compile_source(frozen_graph_model)
+    return tf_onnx_compiler.compile_source(frozen_graph_model)
+
+
+def _make_resnet50_onnx_model_file(file_path):
+    device = torch.device('cpu')
+    dummy_input = Variable(torch.randn(1, 3, 224, 224))
+    dummy_input = dummy_input.to(device)
+    model = torchvision.models.resnet50(pretrained=True)
+    resnet50_onnx_model_path = os.path.join(file_path, 'model', 'resnet50.onnx')
+    dir_name = os.path.dirname(resnet50_onnx_model_path)
+    os.makedirs(dir_name, exist_ok=True)
+    torch.onnx.export(model, dummy_input, resnet50_onnx_model_path, verbose=True, keep_initializers_as_inputs=True)
+    return resnet50_onnx_model_path
+
+
+def _make_calibration_dataset_simple(data_path):
+    img_np = np.random.rand(500, 398, 3) * 255.0
+    img = Image.fromarray(img_np.astype(np.uint8))
+    calibration_data_path = os.path.join(data_path, 'calibration_dataset_sample', 'calibration_data.JPEG')
+    dir_name = os.path.dirname(calibration_data_path)
+    os.makedirs(dir_name, exist_ok=True)
+    img.save(calibration_data_path)
+    calibration_data_txt = os.path.join(data_path, 'calibration_data.txt')
+    with open(calibration_data_txt, 'w') as txt_file:
+        txt_file.write('calibration_data.JPEG\n')
+    return calibration_data_path
```

Comment:
make these code simpler and cleaner, for example: `img = Image.fromarray((255.0 * np.random.rand(500, 398, 3)).astype(np.uint8))`, etc.